build-tests-samples/heft-storybook-react-tutorial/config/jest.config.json

```diff
@@ -1,3 +1,17 @@
 {
-  "extends": "@rushstack/heft-jest-plugin/includes/jest-shared.config.json"
+  "extends": "@rushstack/heft-jest-plugin/includes/jest-shared.config.json",
```

Comment:
For followup - stick this in the config from heft-jest-plugin and normalize the output folders.